IonDotnet/Internals/Text/RawTextReader.cs

```diff
@@ -705,7 +705,7 @@ public IEnumerable<SymbolToken> GetTypeAnnotations()
                     }
 
                     var text = symtab.FindKnownSymbol(a.Sid);
```

Comment:
`FindKnownSymbol` should use `a.ImportLocation` now, since that is the authority on symbols with unknown text.

Reply:
Done.